scripts/py_matter_idl/matter_idl/zapxml/handlers/__init__.py

```diff
@@ -12,12 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from matter_idl.matter_idl_types import Idl
```

Comment:
Does is make sense to include other things like `matter_idl` in known first party? I don't know one way or another just asking

Reply:
I think that it would be better to change the namespace of that module. But now the question might be whether it would be worth of changing it to `chip`? Since, the entire project is going to be renamed, maybe we can "correct" the Python module name as well and name it `matter.idl`?

Reply:
It seems that we could "claim" the `matter` namespace and release the `chip`: https://pypi.org/search/?q=matter